src/headers/rules_op.h

```diff
@@ -32,6 +32,7 @@
 #define NOT_SAME_SRCIP      0xffd /* 0xfff - 0x002 */
 #define NOT_SAME_ID         0xffb /* 0xfff - 0x004 */
 #define NOT_SAME_AGENT      0xff7 /* 0xfff - 0x008 */
+#define NOT_SAME_AGENT_ID   0xff3 
```

Comment:
We use the inverse masks when we want to remove some flag activated, or simply to confirm that some check is not going to be evaluated. For example, `NOT_SAME_ID` deactivates `SAME_ID` at bit-level. 
```
  0000 0000 0100 <---- `SAME_ID`
& 1111 1111 1011 <---- `NOT_SAME_ID`
```

Taking this into account, and following the succession, `NOT_SAME_AGENT_ID` (`GLOBAL_FREQUENCY`) should take the value `0x800` and should be below `NOT_SAME_FIELD`.